_data/catalog/index.json

```diff
@@ -0,0 +1,114 @@
+[
+  {
+    "name": "Basics",
+    "description": "Widgets you absolutely need to know before building your first Flutter app.",
+    "priority": "high",
+    "id":"basics"
+  },
+  {
+    "name": "Material Design",
+    "description": "Flutter implements many of the UI components from Google’s Material Design guidelines.",
```

Comment:
Can a category's description contain links? For example, I'd like to link "Material Design guidelines" to material.io/guidelines